packages/website/src/types/common.ts

```diff
@@ -0,0 +1,32 @@
+type infoBox = {
+  key:string,
+  value:string | {
+    k:string
+    v:string
+  }[]
+}[]
+/* eslint-disable camelcase */
+// 不知道后端会不会改字段
+export type CharacterDetail={
+  id: number
+  name: string
+  type: number
+  images: {
+    large: string
+    medium: string
+    small: string
+    grid: string
+  },
+  summary: string
+  locked: boolean
+  infobox: infoBox
+  gender: string
+  blood_type: number
```

Comment:
```suggestion
  bloodType: number
```

属性的话应该用驼峰，看起来这个类型已经在前端上用到了。可以考虑在API函数中做一个转换。